src/gui/codeeditors/qgscodeeditor.h

```diff
@@ -660,6 +674,11 @@ class GUI_EXPORT QgsCodeEditor : public QsciScintilla
     static QMap< QgsCodeEditorColorScheme::ColorRole, QString > sColorRoleToSettingsKey;
 
     static constexpr int MARKER_NUMBER = 6;
+
+    int ctPos;
+    QStringList ct_entries;
+    int ct_cursor;
+    QList<int> ct_shifts;
```

Comment:
Why are these member variables, instead of defined in callTip?

Reply:
No good reason. I just copied the code from qscintilla.h/cpp. But it does create unecessary clutter in the header file. I will move it to the cpp.